src/test/java/seedu/taassist/model/moduleclass/ModuleClassTest.java

```diff
@@ -77,6 +77,6 @@ public void equals_sameModuleClassNameAndSession_returnsTrue() {
 
         // same name and sessions -> returns true
         ModuleClass cs1101sCopy = new ModuleClass(CS1101S.getClassName(), CS1101S.getSessions());
-        assertTrue(cs1101sCopy.isSame(CS1101S));
```

Comment:
We might want to take a deeper look into this to ensure that other classes does not have the same problem.